packages/client/components/GitLabScopingSearchInput.tsx

```diff
@@ -64,13 +65,29 @@ const GitLabScopingSearchInput = (props: Props) => {
   const {queryString} = gitlabSearchQuery
   const isEmpty = !queryString
   const atmosphere = useAtmosphere()
+  const inputRef = useRef<HTMLInputElement>(null)
+  const {viewerId} = atmosphere
+
+  const trackEvent = (eventTitle: string) => {
+    SendClientSegmentEventMutation(atmosphere, eventTitle, {
+      viewerId,
+      meetingId,
+      service: 'gitlab'
+    })
+  }
 
   const onChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const {value} = e.target
+    if (!queryString) {
+      trackEvent('Started Poker Scope search')
```

Comment:
+1 Not a big deal, but the random capitalization seems odd to me as it is inconsistent with 'Selected Poker Scope Project Filter'. Seeing the existing events, these also seem to have random capital letters.

Reply:
True, I've capitalised `search` to make it more consistent